### DISCUSSION TITLE
REQ: Valheim Plus integrated support?

### DISCUSSION THREAD

**Discussion by Akirainblack** [mbround18/valheim-docker#270] [Ideas]
Would you be willing to incorporate Valheim Plus including updating into the container itself?
I've seen other containers with it built in, however I would prefer to stick with your container.
If that means I have to manage it manually, then that's fine but it never hurts to ask. :)

Thank you for your work on this container.

**Comment by mbround18** [comment]
Hey @Akirainblack, 

To answer your question, Yes this is being worked on. Although, we are looking at taking a more generic approach with an ENV variable called `MODS` which will download and detect what it is and where to extract it. Ive noticed with other repositories that they have a flag to install ValhiemPlus directly which hits the github api many times to get the latest release and install it. 

With the method we are working on it gives you, a server owner, the ability to manage the version and what mods are installed. That way is there is a bug with valheim plus or another mod you can simply choose to install a different version or not at all. 

This is to have similar feature parity with [how this minecraft docker does it.](https://hub.docker.com/r/itzg/minecraft-server)


See https://github.com/mbround18/valheim-docker/issues/272
